recipe/meta.yaml

```diff
@@ -8,59 +8,84 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/greykite-{{ version }}.tar.gz
   sha256: 27c74119900b186361cb4ccdd040eaf9383bc41f7b3781706600891df2c7c6e3
+  patches:
+    0001-setup.patch
 
 build:
+  # missing scs package for windows
+  # missing holidays-ext package for s390x
+  skip: True  # [py<310 or py>=312 or win or s390x]
   entry_points:
     - greykite=greykite.cli:main
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
 requirements:
   host:
-    - python >=3.10
+    - python
     - pytest-runner
     - pip
+    - wheel
+    - setuptools
   run:
-    - python >=3.10
+    - python
     - cvxpy >=1.2.1
     - dill >=0.3.1.1
     - holidays <0.25
     - holidays-ext >=0.0.7
     - ipython >=7.31.1
     - matplotlib-base >=3.4.1
-    - numpy >=1.22.0,<1.25.0
+    - numpy >=1.22.0,<1.24.0a0 # Adding upper bound due to use of "old" np.float
     - osqp >=0.6.2
     - overrides >=2.8.0
     - pandas >=1.5.0,<2.0.0
     - patsy >=0.5.1
     - plotly >=4.12.0
-    - pmdarima >=1.8.0,<=1.8.5
-    - pytest >=4.6.5
+    - pmdarima >=1.8.0 # Removing upper bound as it doesn't breat anything
     - pytest-runner >=5.1
```

Comment:
```suggestion
```

not a runtime dependency (see its usage https://pypi.org/project/pytest-runner/)